src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const isQueryIncluded = (field: string) => (
+    field.toLowerCase().includes(query.toLowerCase())
```

Comment:
```suggestion
    field.toLowerCase().includes(query.toLowerCase().trim())
```
<img width="507" alt="image" src="https://user-images.githubusercontent.com/55802630/184820401-3e192e6e-9d4e-40ae-bbf3-fd1c13b17c23.png">

Reply:
thanks)